DOC-2045: Added TINY-9280 to the release notes. (#2755)
tinymce/tinymce-docs: modules/ROOT/pages/6.5-release-notes.adoc

{productname} 6.5 also incorporates the following changes:

=== Elements in the Insert → Link… dialog did not allow screen readers to announce listed items.

> 

Author: ```suggestion
//#TINY-9280

```

{productname} 6.5 also incorporates the following changes:

> === Elements in the Insert → Link… dialog did not allow screen readers to announce listed items.

Reviewer: ```suggestion
=== List items in a combo box were not announced by screen readers.
```
Copy edit of @tiny-ben-tran’s suggested edit.

//#TINY-9280

> For example, the first UI element in the {productname} *Insert → Link…* dialog is a text-entry field labelled *URL*. When this element is navigated to using a screen reader, it is announced as a combobox and pressing the down arrow opens the combobox and presents available options.

Reviewer 2: ```suggestion
For example, the URL text-entry field in the Link dialog is a combobox. When this element is focused with a screen reader, it is announced as a combobox and pressing the down arrow shows the available list items.
```

In previous versions of {productname}, however, none of these items were announced by the screen reader as they were selected, making these items effectively invisible to screen reader users.

> With this update, the {productname} *Insert → Link…* dialog has been updated to properly support screen readers. When the URL combo box is opened, for example, all listed items are now announced by screen readers, as expected.

Reviewer: ```suggestion
With this update, listed items in a combobox can be announced by screen readers, as expected.
```
Copy-edit of @tiny-ben-tran’s suggested edit.

> With this update, the {productname} *Insert → Link…* dialog has been updated to properly support screen readers. When the URL combo box is opened, for example, all listed items are now announced by screen readers, as expected.

Reviewer 2: ```suggestion
With this update, all listed items in the combobox can be announced by the screen readers, making them more accessible.
```

> === `aria-activedescendant` attribute for `Typeahead` input component is updated to the menu item that is highlighted.
> //#TINY-9280
> 
> In previous versions of {productname}, the Link Combobox screen reader feature lacked the capability to announce items within the dropdown. This meant that when users relied on screen readers to navigate through the dropdown list in the URL input combo box within the link dialog, there were no auditory notifications for the listed items.
> 
> As a consequence, this significantly reduced the accessibility of the combo box for users that relying on screen readers.
> 
> To address this issue, a fix was implemented to enable screen readers to announce the highlighted item within the dropdown. With this improvement, when users navigate through the dropdown, the screen reader will now audibly announce the value of the item currently highlighted.

Reviewer: ```suggestion
=== Elements in the Insert → Link… dialog did not allow screen readers to announce listed items.

For example, the first UI element in the {productname} *Insert → Link…* dialog is a text-entry field labelled *URL*. When this element is navigated to using a screen reader, it is announced as a combobox and pressing the down arrow opens the combobox and presents available options.

In previous versions of {productname}, however, none of these available items were announced by the screen reader as they were selected, making these items effectively invisible to screen reader users.

With this update, the {productname} *Insert → Link…* dialog has been updated to properly support screen readers. When the URL combo box is opened, for example, all listed items are now announced by screen readers, as expected.
```

Suggested re-write.
